tools/tree_relabeler/.shed.yml

```diff
@@ -0,0 +1,29 @@
+categories:
+- Text Manipulation 
+description: Relabels the tips of a newick formatted tree. 
+long_description: |
```

Comment:
when using `|` to indicate a block in yaml, you need to indent all the following lines of the block